contracts/AdvancedProviders/MultiWithdrawProvider/MultiWithdrawProvider.sol

```diff
@@ -63,7 +69,7 @@ contract MultiWithdrawProvider is TransactionState, IInnerWithdraw{
             return (type(uint256).max, true);
         }
         uint256 currentVaultId = uniqueVaultIds[iterator - 1];
-        lockDealNFT.copyVaultId(vaultIdToPoolId[currentVaultId], mintedPoolId);
+        lockDealNFT.setVaultIdOfPool(mintedPoolId, currentVaultId);
```

Comment:
can we use the `copyVaultId` ?
```
lockDealNFT.copyVaultId(iterator - 1,mintedPoolId)
```

Reply:
Originally I was `copyVaultId`. But I thought we could remove 1 state variable if can directly set the vaultID of a pool.